hnn_core/dipole.py

```diff
@@ -102,9 +102,13 @@ def average_dipoles(dpls):
         average over the same components in the input list
     """
     # need at least one Dipole to get times
-    if len(dpls) < 2:
-        raise ValueError("Need at least two dipole object to compute an"
-                         " average")
+    if len(dpls) == 1:
+        print('Only one dipole passed in. '
+              'Consider at least two dipole objects to compute an average.')
+        return dpls[0]
+    elif not dpls:
+        raise ValueError("Need at least one dipole object to return a"
+                         " dipole")
```

Comment:
I would instead raise an error in the loop below if any of the objects in the list is not an instance of `Dipole`. Look around in the codebase to see how it's done

Could you also update a test after doing this? So you get familiar with the test suite and how it works?

Reply:
Yea, that makes sense. I'll put the error detection into the loop and update a test.

Reply:
Updated both the loop and the test. Haven't created test cases before so I hope that what I did makes sense. I used the `# average  two identical dipole objects` as a guide for creating the single dipole object average test. Are there docs that explain the standards for developing test cases with examples?